submissions/ZjeniiaSt/html-css-popup/styles.css

```diff
@@ -1,5 +1,8 @@
 :root {
+  --background-color: rgba(229, 229, 229, 1);
```

Comment:
In case when you have `rgba` color scheme and alpha channel with value 1, better use `rgb`)